orchestration/templates/cron-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: clinvar-ingest
+spec:
+  suspend: {{ not .Values.cron.enable }}
+  schedule: '{{ .Values.cron.schedule }}'
+  concurrencyPolicy: Forbid
```

Comment:
Self TODO: Cron workflow support time zones, which should help us understand the cron schedule better